src/components/wallet/manage/KeyRow.vue

```diff
@@ -264,12 +264,8 @@ export default class KeyRow extends Vue {
 
     select() {
         this.activating = true
-        Promise.resolve()
-            .then(() => new Promise((resolve) => setTimeout(resolve, 100)))
-            .then(() => this.$emit('select', this.wallet))
-            .then(() => new Promise((resolve) => setTimeout(resolve, 50)))
-            .then(() => this.$store.dispatch('Platform/updateActiveDepositOffer'))
-            .catch((error) => console.error(error))
+        setTimeout(() => this.$emit('select', this.wallet), 100)
```

Comment:
If you dispatch change of wallet after 100ms then the line below eill run faster than this one but do not forget tha platfomr module has a dependency of wallet module. So please remove setTimeout as it is uneccesary

Reply:
Done